### PR TITLE
feat: signature certificate download

### PR DESCRIPTION
**Pull request by paolo**
This PR adds a new feature that appends a signature certificate to the pdf when user is downloading the pdf.
1. add new helper method to generate the cerficates(appendCertificates) 
2. connect appendCertificates method with onClickDownload button of two entry points
Screenshot 2024-03-01 at 12 05 37 PM

### CODE REVIEW

**Comment by Mythie** [comment]
Nice start!

File: apps/web/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@documenso/trpc": "*",
     "@documenso/ui": "*",
     "@hookform/resolvers": "^3.1.0",
+    "@react-pdf/renderer": "^3.3.8",
```
Comment (dephraiim) [inline apps/web/package.json:25]: we have `react-pdf` already, any reason for this?

Comment (Mythie) [reply apps/web/package.json:25]: This is a different react-pdf 😄 This one is for creating PDF's in the declarative fashion

Comment (paolo) [reply apps/web/package.json:25]: precisely, this PR will be essentially creating PDFs documents.
this particular library is actually recommended by `react-pdf`: https://github.com/wojtekmaj/react-pdf?tab=readme-ov-file#lost

**Comment by paolo** [comment]
Hi @Mythie , This PR should be ready for review

**Comment by Mythie** [comment]
Looking at these now!

**Comment by IgorHorta** [comment]
> Looking at these now!

Hey @Mythie, any updates here ?

**Pull request closed without merging**